coral-trino/src/main/java/com/linkedin/coral/trino/trino2rel/TrinoConvertletTable.java

```diff
@@ -1,42 +0,0 @@
-/**
```

Comment:
Deleting as we only need one convertlet table for all of cora since all SqlNodes that arrive here should already be in Coral IRl, this will be in a future PR.